devel/forge/Makefile

```diff
@@ -0,0 +1,52 @@
+PORTNAME=	forge
+DISTVERSIONPREFIX=	v
+DISTVERSION=	0.2.0
+CATEGORIES=	devel elisp
+PKGNAMESUFFIX=	${EMACS_PKGNAMESUFFIX}
+
+MAINTAINER=	yasu@utahime.org
+COMMENT=	Work with Git forges from the comfort of Magit (Release Version)
```

Comment:
what does this "(Release Version)" mean?

Reply:
Will remove, thanks! This is opposite to the -devel version to be committed soon.